### ISSUE TITLE
No notifications on Pop_OS! default install

### ISSUE THREAD

**Issue by posita** [flathub/org.signal.Signal#335]
Signal version: 5.54.0

``` sh
% uname -v
#202207071639~1659403207~22.04~cb5f582 SMP PREEMPT_DYNAMIC Tue A
% flatpak --version 
Flatpak 1.12.7
% flatpak info --show-permissions org.signal.Signal
[Context]
shared=network;ipc;
sockets=x11;wayland;pulseaudio;
devices=all;
filesystems=xdg-documents;xdg-desktop;xdg-music;xdg-download;xdg-pictures;xdg-videos;xdg-public-share;

[Session Bus Policy]
org.kde.StatusNotifierWatcher=talk
org.freedesktop.Notifications=talk
org.freedesktop.portal.Background=talk
org.freedesktop.portal.Fcitx=talk
org.kde.*=own
```

![screenshot](https://user-images.githubusercontent.com/222581/184737758-7d071b48-2852-4abe-a210-e3c52696a6ae.png)

**Comment by Altonss** [comment]
The current stable version available is 5.58, is this problem still hapenning to you with this latest version?

**Comment by posita** [comment]
Looks good! Closing!
